src/fundus/publishers/base_objects.py

```diff
@@ -59,6 +69,9 @@ def __init__(self, spec: PublisherSpec):
 
         self.source_mapping = source_mapping
 
+    def __str__(self) -> str:
+        return f"{self.publisher_name}"
+
```

Comment:
This looks like a leftover, or am I missing something?

Reply:
This is needed to print the actual names of the publishers when needed. This line causes `Österreichischer Rundfunk` to be printed instead of `AT.ORF`

Reply:
Thanks for the explanation. I think I mixed up classes in GitHub's file view and thought it was part of `PublisherCollectionMeta`.